kong/cli/utils/services.lua

```diff
@@ -13,8 +13,8 @@ _M.STATUSES = {
 -- Services ordered by priority
 local services = {
   require "kong.cli.services.dnsmasq",
-  require "kong.cli.services.nginx",
-  require "kong.cli.services.serf"
+  require "kong.cli.services.serf",
+  require "kong.cli.services.nginx"
```

Comment:
Yep. Although, like the old behavior, if you terminate nginx it will not terminate the other services, but it doesn't matter if terminating nginx means shutting down the container.